messaging/rabbitmq_queue.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, Manuel Sousa <manuel.sousa@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+DOCUMENTATION = '''
+---
+module: rabbitmq_queue
+author: Manuel Sousa
+version_added: "2.0"
+
+short_description: This module manages rabbitMQ queues
+description:
+  - This module uses rabbitMQ Rest API to create/delete queues
+requirements: [ python requests ]
+options:
+    name:
+        description:
+            - Name of the queue to create
+        required: true
+    state:
+        description:
+            - Whether the queue should be present or absent
+            - Only present implemented atm
+        choices: [ "present", "absent" ]
+        required: false
+        default: present
+    login_user:
+        description:
+            - rabbitMQ user for connection
+        required: false
+        default: guest
+    login_password:
+        description:
+            - rabbitMQ password for connection
+        required: false
+        default: false
+    login_host:
+        description:
+            - rabbitMQ host for connection
+        required: false
+        default: localhost
+    login_port:
+        description:
+            - rabbitMQ management api port
+        required: false
+        default: 15672
+    vhost:
+        description:
+            - rabbitMQ virtual host
+        required: false
+        default: "/"
+    durable:
+        description:
+            - whether queue is durable or not
+        required: false
+        choices: [ "yes", "no" ]
+        default: yes
+    auto_delete:
+        description:
+            - if the queue should delete itself after all queues/queues unbound from it
+        required: false
+        choices: [ "yes", "no" ]
+        default: no
+    message_ttl:
+        description:
+            - How long a message can live in queue before it is discarded (milliseconds)
+        required: False
+    auto_expires:
+        description:
+            - How long a queue can be unused before it is automatically deleted (milliseconds)
+        required: false
```

Comment:
I'll add the defaults, one question. When the default is an empty dictionary is it ok to say default: {} ?